dist/index.js

```diff
@@ -676,12 +676,33 @@ class IssuesProcessor {
     getIssues(page) {
         var _a;
         return __awaiter(this, void 0, void 0, function* () {
+            // Compute labels filter.
+            let labels = '';
+            // We can filter issues already when listing them if neither only-pr-labels
+            // nor only-issue-labels are specified or both are the same.
+            if (this.options.onlyPrLabels === '' &&
+                this.options.onlyIssueLabels === '') {
+                labels = this.options.onlyLabels;
+            }
+            else if (this.options.onlyPrLabels == this.options.onlyIssueLabels) {
+                labels = this.options.onlyIssueLabels;
+            }
```

Comment:
(Microoptimizing for code size, this can be:
```js
  if (this.options.onlyPrLabels == this.options.onlyIssueLables) { 
    labels = this.options.onlyIssueLables || this.options.onlyLabels;
  }
```

Feel free to ignore.)

Reply:
This is generated code - does not make much sense to review it :) Also I don't think code size matters here.